### DISCUSSION TITLE
Issue with Flagged Netflix Clone Project on GitHub

### DISCUSSION THREAD

**Discussion by dolkersonam** [community/community#138676] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Recently, I created and deployed a Netflix clone project on GitHub as part of my portfolio. However, I was unaware of the copyright issues associated with replicating Netflix's branding and content. As a result, my project website, along with all other associated projects, has been flagged.

I have now made the necessary changes to the project, ensuring it no longer infringes on any copyrights. Could you please guide me on how to remove the flag from my website after the revisions?
![Screenshot 2024-09-15 103657](https://github.com/user-attachments/assets/955c2367-ec83-4f2b-9196-9f9172992d59)

**Comment by davevad93** [comment]
Hi @dolkersonam ,

The warning is due to a Google security policy regarding potential phishing sites and even if you have fixed it, Google may have preemptively flagged all your hosted repos as "dangerous". To fix this, take a look at the steps shown [here](https://github.com/orgs/community/discussions/30261#discussioncomment-3467665), but make sure that no repos hosted on GitHub Pages clone web pages like `Netflix`, `Facebook`, `Instagram`, `Spotify` etcetera or contain copyrighted intellectual property that isn't yours, otherwise you will be flagged again. 

Check out the discussions below for additional context: 

1. [My deployment pages in github are marked as dangerous sites.. but they are just simple projects like calculator and my portfolio #133058](https://github.com/orgs/community/discussions/133058)
2. [SECURITY ERROR, HELP!!! #136029](https://github.com/orgs/community/discussions/136029#discussioncomment-10431976)

**Reply by dolkersonam** [reply]
Thank you so much @davevad93 .

All the websites are working just fine now.

**Reply by davevad93** [reply]
Hey @dolkersonam , sorry for the delay in replying, glad you got it sorted out, have a nice day 👍.
